--- giflib-4.1.6/configure.ac.bero	2013-01-13 22:15:36.710072313 +0100
+++ giflib-4.1.6/configure.ac	2013-01-13 22:15:43.424990090 +0100
@@ -2,7 +2,7 @@ dnl Process this file with autoconf to p
 AC_INIT(giflib, [4.1.6], [abadger1999@sourceforge.net], giflib)
 AC_CONFIG_SRCDIR([lib/dgif_lib.c])
 AM_INIT_AUTOMAKE([gnu check-news dist-bzip2 -Wall])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS([config.h])
 
 dnl Checks for programs.
 AC_PROG_LIBTOOL
