Index: util/gif2epsn.c
===================================================================
--- util/gif2epsn.c
+++ util/gif2epsn.c	2010-01-04 11:34:03.000000000 +0100
@@ -146,7 +146,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gif2ps.c
===================================================================
--- util/gif2ps.c
+++ util/gif2ps.c	2010-01-04 11:33:18.000000000 +0100
@@ -129,7 +129,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s",VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gif2rgb.c
===================================================================
--- util/gif2rgb.c
+++ util/gif2rgb.c	2010-01-04 11:34:31.000000000 +0100
@@ -108,7 +108,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gif2x11.c
===================================================================
--- util/gif2x11.c
+++ util/gif2x11.c	2010-01-04 11:49:46.000000000 +0100
@@ -143,7 +143,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifasm.c
===================================================================
--- util/gifasm.c
+++ util/gifasm.c	2010-01-04 11:33:41.000000000 +0100
@@ -98,7 +98,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifbg.c
===================================================================
--- util/gifbg.c
+++ util/gifbg.c	2010-01-04 11:35:31.000000000 +0100
@@ -130,7 +130,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifclip.c
===================================================================
--- util/gifclip.c
+++ util/gifclip.c	2010-01-04 11:36:12.000000000 +0100
@@ -92,7 +92,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifclrmp.c
===================================================================
--- util/gifclrmp.c
+++ util/gifclrmp.c	2010-01-04 11:48:17.000000000 +0100
@@ -111,7 +111,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifcolor.c
===================================================================
--- util/gifcolor.c
+++ util/gifcolor.c	2010-01-04 11:46:43.000000000 +0100
@@ -89,7 +89,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifcomb.c
===================================================================
--- util/gifcomb.c
+++ util/gifcomb.c	2010-01-04 11:36:50.000000000 +0100
@@ -93,7 +93,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/giffix.c
===================================================================
--- util/giffix.c
+++ util/giffix.c	2010-01-04 11:37:34.000000000 +0100
@@ -90,7 +90,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifflip.c
===================================================================
--- util/gifflip.c
+++ util/gifflip.c	2010-01-04 11:38:05.000000000 +0100
@@ -104,7 +104,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifhisto.c
===================================================================
--- util/gifhisto.c
+++ util/gifhisto.c	2010-01-04 11:38:35.000000000 +0100
@@ -112,7 +112,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifinfo.c
===================================================================
--- util/gifinfo.c
+++ util/gifinfo.c	2010-01-04 11:47:26.000000000 +0100
@@ -104,7 +104,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	printf("Format: quoted text string used for formating of information\n");
 	printf("  Special characters for various information can be used\n");
Index: util/gifinter.c
===================================================================
--- util/gifinter.c
+++ util/gifinter.c	2010-01-04 11:39:10.000000000 +0100
@@ -98,7 +98,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifinto.c
===================================================================
--- util/gifinto.c
+++ util/gifinto.c	2010-01-04 11:40:00.000000000 +0100
@@ -96,7 +96,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifovly.c
===================================================================
--- util/gifovly.c
+++ util/gifovly.c	2010-01-04 11:40:37.000000000 +0100
@@ -73,7 +73,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifpos.c
===================================================================
--- util/gifpos.c
+++ util/gifpos.c	2010-01-04 11:41:05.000000000 +0100
@@ -90,7 +90,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifrotat.c
===================================================================
--- util/gifrotat.c
+++ util/gifrotat.c	2010-01-04 11:49:16.000000000 +0100
@@ -117,7 +117,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifrsize.c
===================================================================
--- util/gifrsize.c
+++ util/gifrsize.c	2010-01-04 11:41:44.000000000 +0100
@@ -113,7 +113,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/giftext.c
===================================================================
--- util/giftext.c
+++ util/giftext.c	2010-01-04 11:42:17.000000000 +0100
@@ -102,7 +102,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/gifwedge.c
===================================================================
--- util/gifwedge.c
+++ util/gifwedge.c	2010-01-04 11:43:11.000000000 +0100
@@ -94,7 +94,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/icon2gif.c
===================================================================
--- util/icon2gif.c
+++ util/icon2gif.c	2010-01-04 11:46:11.000000000 +0100
@@ -94,7 +94,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/raw2gif.c
===================================================================
--- util/raw2gif.c
+++ util/raw2gif.c	2010-01-04 11:43:49.000000000 +0100
@@ -118,7 +118,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/rgb2gif.c
===================================================================
--- util/rgb2gif.c
+++ util/rgb2gif.c	2010-01-04 11:44:18.000000000 +0100
@@ -111,7 +111,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
Index: util/text2gif.c
===================================================================
--- util/text2gif.c
+++ util/text2gif.c	2010-01-04 11:44:50.000000000 +0100
@@ -107,7 +107,7 @@
     }
 
     if (HelpFlag) {
-	fprintf(stderr, VersionStr);
+	fprintf(stderr, "%s", VersionStr);
 	GAPrintHowTo(CtrlStr);
 	exit(EXIT_SUCCESS);
     }
